packages/components/src/separator/__tests__/Separator.test.js

```diff
@@ -0,0 +1,10 @@
+// @flow
+
+import * as React from 'react';
+import { shallow } from 'react-native-testing-library';
```

Comment:
Why you use `shallow` and not `render`? Is small and atomic component som my point of view render make more sense.

Reply:
🚀